scripts/eval/yamls/mpt_eval.yaml

```diff
@@ -1,16 +1,19 @@
-max_seq_len: 1024
-tokenizer_name: EleutherAI/gpt-neox-20b
+variables:
+  tokenizer_name: EleutherAI/gpt-neox-20b
+  max_seq_len: 1024
```

Comment:
I want to push back on this a bit - I think it will improve the clarity of the yamls if every interpolation variable used in multiple places (e.g. sequence length) is placed in the `variables` section. This is for two reasons:

- Anyone who wants to tweak variables has a single place to look, and there is a single source of truth for important values at the top of the file rather than having them scattered across the yaml.
- All of the interpolation syntax is consistent (e.g. always `${variables.xyz}` rather than sometimes `${xyz}` and other times `${variables.xyz}`).